Product/Production/Gateway/ExchangeManagement/pom.xml

```diff
@@ -0,0 +1,99 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.connectopensource</groupId>
+    <artifactId>Gateway</artifactId>
+    <version>5.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>ExchangeManagement</artifactId>
+  <name>CONNECT Gateway - ${project.artifactId}</name>
+  <packaging>war</packaging>    
+  <modelVersion>4.0.0</modelVersion>
+
+  <dependencies>
+    <!-- Java EE APIs -->
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.servlet</artifactId>
+    </dependency>
+
+    <!-- CONNECT Modules / Web Services -->
+    <dependency>
+      <groupId>org.connectopensource</groupId>
+      <artifactId>CONNECTCoreLib</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.connectopensource</groupId>
+      <artifactId>CONNECTCommonTypesLib</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.connectopensource</groupId>
+      <artifactId>ConnectionManagerCore</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.connectopensource</groupId>
+      <artifactId>ExchangeManagementWebservices</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <!-- Logging -->
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+    </dependency>
+
+    <!-- Testing -->
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
```

Comment:
You're adding testing frameworks as dependencies, but I don't see any JUnit tests. Can either add JUnit tests, or remove this dependency.